pkg/graphviz/traceflow.go

```diff
@@ -309,6 +334,12 @@ func GenGraph(tf *opsv1alpha1.Traceflow) (string, error) {
 
 	senderRst := getNodeResult(tf, isSender)
 	receiverRst := getNodeResult(tf, isReceiver)
+	if tf.Status.Phase != opsv1alpha1.Succeeded {
+		err := getTraceflowStatusMessage(tf, graph)
```

Comment:
it doesn't seem that the function will return an error, and having a side effect in a get function is obscure, maybe just let the function return the message and set the label in `GenGraph`, which could be more consistent as well?

Reply:
Function modified.